src/main/java/commands/FindCommand.java

```diff
@@ -0,0 +1,36 @@
+package commands;
+
+import java.util.ArrayList;
+
+import handlers.Ui;
+import handlers.Storage;
+
+import tasks.Task;
+import tasks.TaskList;
+
+public class FindCommand extends Command {
+    ArrayList<Task> helper = new ArrayList<>();
```

Comment:
You can consider using the List interface instead of ArrayList in case you decide to change to a different type of list.